src/portable/synopsys/dwc2/dwc2_stm32.h

```diff
@@ -56,6 +56,11 @@
   #define EP_FIFO_SIZE_FS 4096
   #define EP_MAX_HS       9
   #define EP_FIFO_SIZE_HS 4096
+  #if (! defined USB2_OTG_FS)
+    // H7 with only 1 USB port: H72x / H73x / H7Ax / H7Bx
+    // USB_OTG_FS_PERIPH_BASE and OTG_FS_IRQn not defined
+    #define USB_OTG_FS_PERIPH_BASE  USB1_OTG_HS_PERIPH_BASE
+    #define OTG_FS_IRQn             OTG_HS_IRQn
```

Comment:
seems like you miss an #endif, please make sure you compile the example and verify the changes works

Reply:
Sorry this is my first PR, and I got confused.
I'm sure it works now.